File: src/com/charter/enterprise/collatz/util/OddEvenDeterminer.java
package com.charter.enterprise.collatz.util;

public class OddEvenDeterminer {

    public boolean isOdd(long potentialOdd) {
        if(potentialOdd % 2 != 0) {
            return true;
        }

        return false;
    }

    public boolean isEven(long potentialEven) {
        return !isOdd(potentialEven);
    }
}

File: src/util/odd-even-determiner.ts
export class OddEvenDeterminer {
    isOdd(potentialOdd: number): boolean {
        if (potentialOdd % 2 !== 0) {
            return true;
        }
        return false;
    }

    isEven(potentialEven: number): boolean {
        return !this.isOdd(potentialEven);
    }
}
